packages/rempl-apollo-devtools/apollo-devtools-maglev.css

```diff
@@ -0,0 +1,4 @@
+body > div[side="top"],
+body > div[side="right"] {
+  top: 40px;
```

Comment:
Why not 42? 🤔

Reply:
Because it was just enough when i was testing it in maglev